frontend/src/pages/PipelineDetails.test.tsx

```diff
@@ -203,10 +203,10 @@ describe('PipelineDetails', () => {
     },
   );
 
-  it('parses the workflow source in embedded pipeline spec as JSON and then converts it to YAML', async () => {
+  it('use the pipeline manifest in embedded pipeline spec as template string', async () => {
     testRun.run!.pipeline_spec = {
       pipeline_id: 'run-pipeline-id',
-      workflow_manifest: '{"spec": {"arguments": {"parameters": [{"name": "output"}]}}}',
+      pipeline_manifest: 'spec:\n  arguments:\n    parameters:\n      - name: output\n',
     };
```

Comment:
I think this test is supposed to test v1 pipeline, not v2 pipeline. So we cannot change from `workflow_manifest` to `pipeline_manifest`